google/auth/impersonated_credentials.py

```diff
@@ -173,6 +180,7 @@ def __init__(
         lifetime=_DEFAULT_TOKEN_LIFETIME_SECS,
         quota_project_id=None,
         iam_endpoint_override=None,
+        universe_domain=None,
```

Comment:
as per GUAC discussion, do not introduce a way to set universe_domain in top level cred, if not already there.

Reply:
Yep, that was older impelementation